e2e/porch_test.go

```diff
@@ -70,6 +71,7 @@ func runTestCase(t *testing.T, git string, tc porch.TestCaseConfig) {
 	}
 
 	for i := range tc.Commands {
+		time.Sleep(5 * time.Second)
```

Comment:
k8s apiserver seems to be throttling and rejecting requests, hence this timeout.
It is a temporary solution but it reduces flakiness.